evaluate.js

```diff
@@ -0,0 +1,85 @@
+Array.prototype.clean = function() {
+    for(let i = 0; i < this.length; i++) {
+        if(this[i] === "") {
+            this.splice(i, 1);
+        }
+    }
+    return this;
+}
+
+function expressionToRpn(expression) {
+        let outputQueue = "";
+        let operatorStack = [];
+        let operators = {
+            "^": {
+                precedence: 4,
+                associativity: "Right"
+            },
+            "/": {
+                precedence: 3,
+                associativity: "Left"
+            },
+            "*": {
+                precedence: 3,
+                associativity: "Left"
+            },
+            "+": {
+                precedence: 2,
+                associativity: "Left"
+            },
+            "-": {
+                precedence: 2,
+                associativity: "Left"
+            }
+        }
+        expression = expression.replace(/\s+/g, "");
+        expression = expression.split(/([\+\-\*\/\^\(\)])/).clean();
+        for(let i = 0; i < expression.length; i++) {
```

Comment:
please use build-in functions
forEach here

Reply:
changed as recommended